cyhy/mailer/cli.py

```diff
@@ -1060,6 +1145,32 @@ def main():
         )
         return 1
 
+    # Parse out the CSA email mapping, if present.
+    csa_email_file = args["--csa-email-file"]
+    if csa_email_file is not None:
+        try:
+            with open(csa_email_file) as stream:
+                csa_email_mapping = yaml.load(stream, Loader=yaml.SafeLoader)
+        except OSError:
+            logging.critical(
+                f"CSA email file {csa_email_file} does not exist", exc_info=True
+            )
+            return 1
+        except yaml.YAMLError:
+            logging.critical(
+                f"CSA email file {csa_email_file} does not contain valid YAML",
+                exc_info=True,
+            )
+            return 1
+
+    # Rearrange the CSA email mapping data into a more convenient
+    # form.
+    csa_emails = {}
+    for region in csa_email_mapping:
```

Comment:
If `csa_email_file` is `None`, won't `csa_email_mapping` will be undefined here?

(I agree with you that the `--csa-email-file` flag should be optional; if it is not provided, then we should just use the default BCC list.)

Reply:
Please see commit 56b729e4f7a7432e2a0faae37bad1d0ea6a3aaca.

Reply:
Please see commit [ffccc70](https://github.com/cisagov/cyhy-mailer/pull/101/commits/ffccc70d93e614d8e4dbe5eb7ff8ae77fce8acfe).  I tested by doing:
- A full CyHy reports dry run where the CSA email YAML was specified
- A full CyHy reports dry run without specifying the CSA email YAML

Reply:
Yes, that is the cacafuego that I'm talking about! 🌶️ 💩 🔥 
Thanks for making it happen.